Base/QTGUI/qSlicerExportNodeDialog.cxx

```diff
@@ -357,7 +357,9 @@ QString qSlicerExportNodeDialogPrivate::defaultFilename(vtkMRMLNode* node, QStri
     qCritical() << Q_FUNC_INFO << "failed: Core IO manager not found.";
     return QString();
   }
-  const QString safeNodeName = qSlicerCoreIOManager::forceFileNameValidCharacters(unsafeNodeName);
+
+  QString safeNodeName = qSlicerCoreIOManager::forceFileNameValidCharacters(unsafeNodeName);
+  safeNodeName = qSlicerCoreIOManager::forceFileNameMaxLengthExtension(safeNodeName, 0);
   return forceFileNameExtension(safeNodeName, extension, node);
```

Comment:
```suggestion
  return qSlicerExportNodeDialogPrivate::forceFileNameExtension(safeNodeName, extension, node);
```

Reply:
Thanks @jcfr, makes sense to me. Changes made.